Algorithm.Framework/Alphas/MacdAlphaModel.cs

```diff
@@ -111,6 +111,10 @@ public override void OnSecuritiesChanged(QCAlgorithmFramework algorithm, Securit
         {
             foreach (var added in changes.AddedSecurities)
             {
+                if (_symbolData.ContainsKey(added.Symbol))
```

Comment:
Can you give some details about why this appears to be adding a second time?

Reply:
When the security is removed from the universe, we don't remove the `SymbolData`:
```csharp
foreach (var removed in changes.RemovedSecurities)
{
    SymbolData data;
    if (_symbolData.TryGetValue(removed.Symbol, out data))
    {
        // clean up our consolidator
        algorithm.SubscriptionManager.RemoveConsolidator(data.Security.Symbol, data.Consolidator);
    }
}
```
Now I realized that, if the dictionary contains the key, we need to add the consolidator again, so we should use `TryRemove` below.